typescript/packages/subsurface-viewer/src/layers/wells/wellsLayer.stories.tsx

```diff
@@ -415,6 +454,122 @@ AllWellHeadsHidden.parameters = {
     },
 };
 
+const testWell: FeatureCollection = {
+    type: "FeatureCollection",
+    features: [
+        {
+            type: "Feature",
+            geometry: {
+                type: "GeometryCollection",
+                geometries: [
+                    {
+                        type: "Point",
+                        coordinates: [0, 0],
+                    },
+                    {
+                        type: "LineString",
+                        coordinates: [
+                            [0, 0, 0],
+                            [0, 0, 1],
+                            [0, 0, 2],
+                            [0, 50, -50],
+                            [0, 0, -100],
+                            [99, 99, -150],
+                            [99, 0, -250],
+                        ],
+                    },
+                ],
+            },
+            properties: {
+                name: "well99",
+                color: [255, 255, 0, 255],
+                md: [[0, 1, 2, 3, 4, 5, 8, 9]],
+            },
+        },
+    ],
+};
+
+const BBox = [-100, -100, -250, 100, 100, 0] as [
+    number,
+    number,
+    number,
+    number,
+    number,
+    number,
+];
+
+export const WellsRefine: StoryFn<typeof SubsurfaceViewer> = (args) => {
+    const [refineNumber, setRefineNumber] = React.useState<number>(1);
+
+    const props = {
+        ...args,
+        layers: [
+            {
+                "@@type": "WellsLayer",
+                data: testWell,
+                refine: refineNumber,
+            },
+            {
+                "@@type": "AxesLayer",
+                ZIncreasingDownwards: false,
+                bounds: BBox,
+            },
+        ],
+    };
+
+    const handleChange = React.useCallback(
+        (_event: Event | SyntheticEvent, value: number | number[]) => {
+            setRefineNumber(value as number);
+        },
+        []
+    );
+
+    return (
+        <Root>
+            <div className={classes.main}>
+                <SubsurfaceViewer {...props} />
+            </div>
+            <Slider
+                min={1}
+                max={10}
+                defaultValue={1}
+                step={1}
+                onChange={handleChange}
+                valueLabelDisplay={"auto"}
+            />
+        </Root>
```

Comment:
There's a lot of wasted vertical space between the story and the controls, but I guess it does the job

![image](https://github.com/equinor/webviz-subsurface-components/assets/19621822/9143784d-030e-4d57-8cc7-31a3eaf4549b)